background.js

```diff
@@ -201,6 +201,7 @@ function empty_queue(){
     // removeCertainVideo(0);
     main.songs.splice(0,1);
   }
+  chrome.browserAction.setBadgeText({text: left.toString()});
```

Comment:
You need to change the background color before this line to keep it consistent.
`chrome.browserAction.setBadgeBackgroundColor({ color: [0, 0, 0, 255] });`

Also, did you try running it after making the changes in developer mode ?